tests/test_simple.py

```diff
@@ -21,6 +21,14 @@ def testEnvironment(self):
         self.assertIn(envVar, os.environ)
         self.assertTrue(os.path.exists(os.environ[envVar]), f"Check path {os.environ[envVar]}")
 
+    def testNoImplicitMultithreading(self):
+        """Test that the environment has turned off implicit
+        multithreading.
+        """
+        envVar = "OMP_NUM_THREADS"
```

Comment:
Is there anything you could do in a test to try to spawn extra threads and confirm that it doesn't?

Reply:
So I don't know exactly that, but I did add a test to check the number of threads that numexpr thinks it has (following `lsst.utils`)